docs/prod-install.md

```diff
@@ -60,7 +60,17 @@ fail.  Also note that it automatically logs a transcript to
 `/var/log/zulip/install.log`; please include a copy of that file in
 any bug reports.
 
-## Step 3: Configure Zulip
+## Step 4: Set up email on your host
```

Comment:
I ended up being pretty happy with this being its own step just after "Configure Zulip".